index.d.ts

```diff
@@ -1620,7 +1620,7 @@ declare namespace Eris {
     reactionButtonTimeout: number;
     requirements: CommandRequirements;
     restartCooldown: boolean;
-    subcommandAliases: { [alias: string]: Command };
+    subcommandAliases: { [alias: string]: string };
```

Comment:
could `Record<string, string>` be used instead?

Reply:
(on mobile) is it used anywhere else?

Reply:
It turns out we do a mix of both for some reason